### DISCUSSION TITLE
[FEATURE] does anyone know how can i add exception for firefox so the decorations dont apply on it (like opacity )

### DISCUSSION THREAD

**Discussion by theabsoluteray** [mylinuxforwork/dotfiles#576] [General]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by ferr0** [comment]
In the wiki for Hyprland, there is a variable called "windowrule" for window rules.
Link: https://wiki.hyprland.org/Configuring/Window-Rules/

conf: ```.config/hypr/conf/windowrules/default.conf```

For example, such parameters:
```
windowrulev2 = opacity 1.00 1.00,class:^(firefox)$
```
